packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -149,16 +155,17 @@ describe('babel-plugin-strip-runtime using source code', () => {
         `);
       });
 
-      it('calls onFoundStyleRules for every found style', () => {
-        const onFoundStyleRules = jest.fn();
+      it('adds require statement for every found style', () => {
         const baked = transform(code, { run: 'bake', runtime });
-
-        transform(baked, { onFoundStyleRules, run: 'both', runtime });
-
-        expect(onFoundStyleRules).toHaveBeenCalledTimes(1);
-        expect(onFoundStyleRules).toHaveBeenCalledWith([
-          '._1wyb1fwx{font-size:12px}',
-          '._syaz13q2{color:blue}',
+        const actual = transform(baked, {
+          styleSheetName: testStyleSheetName,
+          run: 'extract',
```

Comment:
I updated this the be `extract` instead of `both` because this test falls under the `when run in subsequent steps` describe block, so I was pretty sure it was meant to bake and extract separately - we run bake on line 159 so I assumed this was meant to be extract, but let me know if I am incorrect.